geode-docs/managing/security/implementing_authorization.html.md.erb

```diff
@@ -232,6 +232,8 @@ This table classifies the permissions assigned for JMX operations.
 | CacheServerMXBean.closeAllContinuousQuery      | CLUSTER:MANAGE:QUERY      |
 | CacheServerMXBean.closeContinuousQuery         | CLUSTER:MANAGE:QUERY      |
 | CacheServerMXBean.executeContinuousQuery       | DATA:READ                 |
+| CqQuery.execute                    | DATA:READ:RegionName and CLUSTER:MANAGE:QUERY  |
+| CqQuery.executeWithInitialResults  | DATA:READ:RegionName and CLUSTER:MANAGE:QUERY  |
```

Comment:
CQs cannot be created or executed from gfsh, so we should get rid of these two lines.